dashboards/google-kubernetes-engine/README.md

```diff
@@ -46,3 +46,12 @@
 |:---------------------|
 |Filename: [gke-nodes-pods-cluster-view.json](gke-nodes-pods-cluster-view.json)|
 |This dashboard has 10 charts using the related [Kubernetes metrics](https://cloud.google.com/monitoring/api/metrics_kubernetes) to show how pods and containers are distributed within nodes and within namespaces. Container start, restart and autoscaler events are also plotted over time.|
+
+&nbsp;
+
+|GKE Optimization Golden Signals|
```

Comment:
Can you add an entry for the new dashboard to the metadata.yaml file in the same directory?

Reply:
done